mappings/net/minecraft/entity/effect/StatusEffect.mapping

```diff
@@ -10,6 +10,8 @@ CLASS net/minecraft/class_1291 net/minecraft/entity/effect/StatusEffect
 		ARG 0 type
 	METHOD method_5555 onApplied (Lnet/minecraft/class_1309;Lnet/minecraft/class_1325;I)V
 	METHOD method_5556 getColor ()I
+	METHOD method_5559 getOrCreateTranslationKey ()Ljava/lang/String;
```

Comment:
A translation key isn't "created". Even though it is lazily computed, we can still just call it 
```suggestion
	METHOD method_5559 getTranslationKey ()Ljava/lang/String;
```